MultiTable/src/main.js

```diff
@@ -248,7 +248,7 @@ function setSampleListeners(subjectGroup) {
       }
 
       if (s.relatedLinks && s.relatedLinks.length > 1) {
-        s.relatedLinks.sort(Utils.sort);
+        s.relatedLinks.sort(Utils.sortByNameAscending);
```

Comment:
@iamigo I followed the steps listed in the readme to create the zip file. I am done with compiling the lens. I will talk to you about the build step.